docs/CLI.md

```diff
@@ -224,6 +224,10 @@ This feature is an escape-hatch. If Jest doesn't exit at the end of a test run,
 
 Show the help information, similar to this page.
 
+### `--ignoreProjects <project1> ... <projectN>`
```

Comment:
what happens if both this and `selectProjects` is used? Ideally it should be a configuration error. (and be mentioned in the docs)

Reply:
`selectProjects` and `ignoreProjects` could be used together. now it seems no need to do it because we don't support glob pattern, but I think to allow them to be used together will be better because we maybe support glob in both `selectProjects` and `ignoreProjects`.

and I don't think there is any reason to restrict our user to use them together, If they like to, we will allow them to use.

Reply:
see how our user's opinion here: https://github.com/facebook/jest/issues/12618#issuecomment-1084831167

Reply:
@F3n67u Glob support for both would be awesome as well, especially if you're using https://github.com/jest-community/jest-runner-eslint so you could do ignore *-test when running tests and ignore *-lint when running lint.

Reply:
Mind opening up a separate feature request?